src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas3/ReportsController.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.controller.cas3
+
+import org.springframework.core.io.InputStreamResource
+import org.springframework.core.io.Resource
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.cas3.ReportsCas3Delegate
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.ServiceName
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.BadRequestProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.ForbiddenProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.reporting.properties.TransitionalAccommodationReferralReportProperties
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.UserAccessService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.cas3.ReportService
+import java.io.ByteArrayOutputStream
+import java.util.UUID
+
+@Service("Cas3ReportsController")
+class ReportsController(
```

Comment:
I thought about it, but i didnt changed it because the controller class is kept under CAS3 pkg which implies its for CAS3 and most of the existing controllers are kept in the same way. hence i didnt changed it. But i see your point and confusion.

Reply:
Agreed, being consistent makes sense. I'll make a note for the next cross CAS call and maybe we can do a bulk change if we all like the idea